buildkite/bazelci.py

```diff
@@ -1968,11 +1968,10 @@ def try_update_last_green_commit():
 
     # Find any failing steps other than Buildifier and "try update last green".
     def HasFailed(job):
-        # Some irrelevant job doesn't have "state" field.
-        if "state" not in job:
-            return True
+        state = job.get("state")
+        # Ignore steps that don't have a state (like "wait").
         return (
-            job["state"] != "passed"
+            state != None and state != "passed"
```

Comment:
Then "HasFailed" will return False for irrelevant job, which will block updating last green commit?

Reply:
Oh, sorry. I was stupid..